site/content/docs/0.0/layout/grid.md

```diff
@@ -1,11 +1,632 @@
 ---
 layout: docs
 title: Grid system
-description: Use our powerful mobile-first flexbox grid to build layouts of all shapes and sizes thanks to a twelve column system, six default responsive tiers, Sass variables and mixins, and dozens of predefined classes.
+description: Use our powerful mobile-first flexbox grid to build layouts of all shapes and sizes thanks to a twelve column system, eight default responsive tiers, Sass variables and mixins, and dozens of predefined classes.
 group: layout
 aliases:
   - "/docs/layout/grid/"
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Example
+
+OUDS Web's grid system uses a series of containers, rows, and columns to layout and align content. It's built with [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) and is fully responsive. Below is an example and an in-depth explanation for how the grid system comes together.
+
+{{< callout info >}}
+**New to or unfamiliar with flexbox?** [Read this CSS Tricks flexbox guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flexbox-background) for background, terminology, guidelines, and code snippets.
+{{< /callout >}}
+
+{{< example class="bd-example-row" >}}
+<div class="container text-center">
```

Comment:
Since we can't use `.container`, shouldn't we modify all the examples by using `.container-fluid`?